examples/editor/src/App.tsx

```diff
@@ -13,7 +13,7 @@ function App() {
     },
     domAttributes: {
       editor: {
-        class: styles.editor,
+        class: "editor",
```

Comment:
if we're not using CSS modules, we need to have a good strategy to prevent collisions, for example, prefix everything with `bn-`